internal/corazawaf/waf.go

```diff
@@ -270,6 +273,7 @@ func NewWAF() *WAF {
 		logger.Error("error creating serial log writer: %s", err.Error())
 	}
 	waf := &WAF{
+		transactionPool:          sync.NewPool(func() interface{} { return new(Transaction) }),
```

Comment:
This is particularly an important change as different instances of WAF would reuse the same pool of transactions which would end up on reusing the body buffer settings of the first one declared as in https://github.com/M4tteoP/coraza/pull/1/files#diff-b7d81193324b35778d3c6feb2aa00b46357803142e09e27549afcae2c8b47f0cR182

cc @jptosso @anuraaga

Reply:
Can this be split off of this PR?

Reply:
Done https://github.com/corazawaf/coraza/pull/558